.github/workflows/firebase-hosting-merge.yml

```diff
@@ -0,0 +1,20 @@
+# This file was auto-generated by the Firebase CLI
+# https://github.com/firebase/firebase-tools
+
+name: Deploy to Firebase Hosting on merge
+'on':
+  push:
+    branches:
+      - master
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: yarn build:web
```

Comment:
Somewhere here we need `yarn` command to install all the packages. 
I see the build failed on react scripts probably because there are no packages installed.